reframe/core/schedulers/slurm.py

```diff
@@ -193,6 +197,17 @@ def _get_reservation_nodes(self):
         node_descriptions = completed.stdout.splitlines()
         return (SlurmNode(descr) for descr in node_descriptions)
 
+    def _get_excluded_nodes(self):
+        command = 'scontrol show -o -a %s' % self.sched_exclude_nodelist
+        completed = os_ext.run_command(command, check=True)
+
+        node_descriptions_output = completed.stdout()
```

Comment:
The `stdout` is a property of `completed`, not a function. I don't know how this has worked... Also you can use it directly in the following if.